src/collective/nitf/tests/test_content.robot

```diff
@@ -23,9 +23,9 @@ Test Folder Full View
     Open Add New Menu
     Click Link  css=a#folder
     Page Should Contain  Add Folder
-    Input Text  css=#form-widgets-IDublinCore-title  Test Folder
+    Input Text  css=#form-widgets-IDublinCore-title,#title  Test Folder
```

Comment:
Do robots accepts all css selectors? Did you try this or is this trick in the documentation?